include/fem-dem/gls_sharp_navier_stokes.h

```diff
@@ -497,6 +532,8 @@ Return a bool that describes  if a cell contains a specific point
    */
 
 private:
+  bool all_spheres = true;
```

Comment:
```suggestion
  bool all_spheres;
```

You fixed the default value in the constructor, so you don't need to put a value here. It's not a good idea to have the values in the .h.